src/main/java/com/github/charithe/kafka/KafkaJunitRule.java

```diff
@@ -84,8 +90,12 @@ public KafkaJunitRule(final int kafkaPort) {
 
     @Override
     protected void before() throws Throwable {
-        zookeeper = new TestingServer(true);
-        zookeeperPort = zookeeper.getPort();
+        if (zookeeperPort == -1) {
```

Comment:
I prefere to put a new constant, because in fact the port for zookeeper is not random but assigned by the server...